web/src/features/panels/zone/util.ts

```diff
@@ -44,15 +45,16 @@ export const getZoneDataStatus = (
   // If there is no config for the zone, we assume we do not have any data
   const zoneConfig = config.zones[zoneId];
   if (!zoneConfig) {
-    console.log(zoneConfig);
-
```

Comment:
you can potentially hide this in a
`if (import.meta.env.DEV){}` clause instead if you want it for debug builds.

Reply:
It was an old left-over console log that seemed like a mistake, so I just removed it :)